config/runner.yml

```diff
@@ -22,6 +22,7 @@ drupal:
   root: "build"
   root_absolute: ~
   base_url: "http://127.0.0.1:8888"
+  settings_local_file: 'settings.override.php'
```

Comment:
Let's also pass this one as an overridable parameter to the command.

Reply:
Do you mean in this file?: https://github.com/openeuropa/task-runner/pull/82/files#diff-9a1d65d4e37665bfc2f924f6cc5c8c4e
It's done already. Do I have to modify other files?

Reply:
Done: https://github.com/openeuropa/task-runner/pull/82/files#diff-9a1d65d4e37665bfc2f924f6cc5c8c4e